closed/NVIDIA/code/stable-diffusion-xl/tensorrt/builder.py

```diff
@@ -344,7 +344,7 @@ def __init__(self,
                  # TODO: Legacy value - Remove after refactor is done.
                  precision: str,  # config precision only sets the UNetXL precision
                  config_ver: str = "default",
-                 model_path: str = "build/models/SDXL/",
+                 model_path: str = "models/SDXL/",
```

Comment:
we normally store the models inside build folder only. I think we should use that.

Reply:
Reverted.